examples/distributed/README.md

```diff
@@ -0,0 +1,63 @@
+# Launch distributed DeepGNN in cluster
+
+## Using ansible
+
+### Preparation
+
+1. Linux nodes
+
+    a) Setup SSH: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/ssh-from-windows
+
+    b) Go to `Inventory` folder and add your linux servers in `linux` section:
+
+    ```YAML
+    linux:
+      hosts:
+        azure-linux-vm01: # machine alias
+          ansible_host: x.x.x.x # machine ip
+      vars:
+        ansible_connection: paramiko
+        ansible_user: azureuser # ssh user
+        ansible_ssh_pass: "password" # ssh password
+    ```
+
+2. Windows nodes
+
+    a) Setup windows powershell remote: https://docs.ansible.com/ansible/latest/user_guide/windows.html
+
+    b) Go to `Inventory` folder and add your linux servers in `windows` section:
+
+    ```YAML
+    windows:
+      hosts:
+        azure-win-vm02: # machine alias
+          ansible_host: x.x.x.x # machine ip
+      vars:
+        ansible_connection: psrp
+        ansible_psrp_protocol: http
+        ansible_port: 5985
+        ansible_user: azureuser # windows user
+        ansible_password: "password"
+        ansible_python_interpreter: python
+    ```
+
+### Usage
+
+Go to `distributed/ansible` folder and invoke following command:
+
+```Shell
+python distributed.py \
```

Comment:
there is a typo here, `distribute.py` instead of `distributed.py`.
How do we make sure this document is in sync with the python file?

Reply:
Good catch! Thanks!

Currently I copied the commands in `run.sh` to README.md to make sure they are the same (at least these commands are runnable in run.sh). At this time let me do it manually. Maybe next step we can do it using notebook. Does it make sense?